web/vtadmin/src/components/dropdown/Dropdown.tsx

```diff
@@ -54,7 +65,7 @@ const Dropdown: React.FC<DropdownProps> = ({ children, dropdownButton, position
                 <Menu.Items
                     className={`py-2 z-10 origin-top-right absolute ${
                         positions[position as string] || positions.default
-                    } md:-left-3full mt-2 w-max rounded-lg shadow-lg bg-white ring-1 ring-black ring-opacity-5 divide-y divide-gray-100 focus:outline-none`}
+                    } mt-2 w-max rounded-lg shadow-lg bg-white ring-1 ring-black ring-opacity-5 divide-y divide-gray-100 focus:outline-none`}
```

Comment:
Removed `md:-left-3full` from here, not sure why this was added. Even for "bottom-right" position, the dropdown menu was rendered at "bottom-left".